rest-api/test_backend.py

```diff
@@ -8,31 +8,7 @@ def test_get_leaderboard():
     Test the functionality of the GET request
     """
     resp = app.test_client().get('/leaderboard')
-    expected = [
-      {
-        "_id": "602cf1530fee0fa57f2ef5f7",
-        "name": "Nam",
-        "score": 31223.0
-      },
-      {
-        "_id": "602ce6890fee0fa57f2ef5f6",
-        "name": "Michael",
-        "score": 29943.0
-      },
-      {
-        "_id": "602d118a6fc98b2990e2b682",
-        "name": "Edward",
-        "score": 27181
-      },
-      {
-        "_id": "602d11996fc98b2990e2b683",
-        "name": "Brett",
-        "score": 19044
-      },
-    ]
-
     assert resp.status_code == 200
-    assert expected == resp.get_json()["leaderboard"][:4]
 
 def test_post_delete_leaderboard():
```

Comment:
Since the content of the leaderboard in the database usually changes, It is quite difficult to test for the GET method and see if GET retrieve the correct data.  For the improved Unit Test for the GET method, I first POST a few high scores to the leaderboard on the database, and test to see if GET correctly retrieve those high scores. After, I DELETE any scores I just added to preserve the old content of the leaderboard. You can check these changes under the last commit, "Update Backend Testing for GET requests. Delete blank lines in Form.js"